src/main/scala/shine/DPIA/primitives/functional/AsScalar.scala

```diff
@@ -12,19 +12,21 @@ import shine.DPIA._
 import scala.xml.Elem
 
 final case class AsScalar(
-  n: Nat,
-  m: Nat,
-  dt: ScalarType,
-  access: AccessType,
-  array: Phrase[ExpType]
-) extends ExpPrimitive {
+                           n: Nat,
+                           m: Nat,
+                           dt: ScalarType,
+                           access: AccessType,
+                           array: Phrase[ExpType]
+                         ) extends ExpPrimitive {
```

Comment:
I am not a fan of all these indentation changes. The deleted code is more readble to me and takes less screen width.

Reply:
I have reverted these changes in the latest commit.